File: src/test/java/io/bloco/faker/components/AppTest.java
package io.bloco.faker.components;

import org.junit.Before;
import org.junit.Test;

import io.bloco.faker.Faker;

import static org.junit.Assert.assertNotNull;

public class AppTest {

    private Faker faker;

    @Before
    public void setUp() {
        faker = new Faker();
    }

    @Test
    public void name() {
        assertNotNull(faker.app.name());
    }

    @Test
    public void version() {
        assertNotNull(faker.app.version());
    }

    @Test
    public void author() {
        assertNotNull(faker.app.author());
    }
}

File: lib/src/test/kotlin/io/bloco/faker/components/AppTest.kt
package io.bloco.faker.components

import io.bloco.faker.Faker
import junit.framework.TestCase.assertNotNull
import org.junit.Test

class AppTest {
    private val faker = Faker()

    @Test
    fun name() {
        assertNotNull(faker.app.name())
    }

    @Test
    fun version() {
        assertNotNull(faker.app.version())
    }

    @Test
    fun author() {
        assertNotNull(faker.app.author())
    }
}